Doc/library/pdb.rst

```diff
@@ -74,6 +74,7 @@ The debugger's prompt is ``(Pdb)``, which is the indicator that you are in debug
    command arguments, e.g. the current global and local names are offered as
    arguments of the ``p`` command.
 
+.. _pdb-cli:
```

Comment:
```suggestion
.. _pdb-cli:

Command-line interface
----------------------
```

And there's just one other header in this file, let's also change that to "Debugger Commands" -> "Debugger commands".